### ISSUE TITLE
local.wordpress.dev is not set up to accept connections on port

### ISSUE THREAD

**Issue by txbredred** [Varying-Vagrant-Vagrants/VVV#550]
I had some trouble getting a local connection, but somehow it finally resolved itself. So now when I type in 192.168.50.4 I get the links for the four local WP sites; however, when I click on them it will not display the page. I get the following alert after I click on the Fix connection problems and I do not know how/where to resolve this.

The device or resource won't accept the connection.
The device or resource (local.wordpress.dev) is not set up to accept connections on port "The World Wide Web service (HTTP)"

I am using VirtualBox for my VM. Let me know if there is any additional info I need to provide. 

I am at a loss. I have been trying to set up VVV for 3 days now w/o success b/c I keep running into different issues. Hopefully this is the last thing to resolve.

**Comment by cfoellmann** [comment]
Sounds like you do not have the `vagrant-hostsupdater` installed?

Please check that

**Comment by txbredred** [comment]
I had the same thought, but I downloaded/unzipped both the hostsupdater and triggers plugins and installed both via my CLI (Git Bash).

Does it matter which directory I'm in when I type in the vagrant plugin install command?

Since my OP I uninstalled Vagrant and VirtualBox and trying a new install from scratch. I have not cloned VVV as of yet. I'm confused on whether I need to follow any of the Getting Started instructions from the Vagrant website before proceeding with VVV or does cloning VVV automatically install what I need to get started?

Additional, somewhat unrelated question:
I'm on Windows 7 and installed Git/GitHub for Windows and using Git Bash for my CLI. When I use Git Bash, do I use the same commands as a Linux user?

This is all new to me, but I am determined to get this! And I'm good at following good directions. Thanks for replying.

**Comment by jerome-nelson** [comment]
@txbredred

** plugins
Nope. You can install plugins from any directory. If you think about the fact that the plugin installs are for more than one project it's easier to remember.

** Git Bash
Git/Github for Windows Command Line is a BASH Emulator  of *NIX [Linux, Unix etc...] git.

So yup Linux commands all the way (I also use Git Bash btw and it's fast becoming my do-all of choice).

Are you still gettng the host error message? (Make sure you have permissions to edit the HOSTS file in Windows/system3/etc/ as when I use `vagrant up` my anti-virus blocks access).

**Comment by txbredred** [comment]
Yes, still getting the host error.

sh: line 5: C:/Windows/system32/drivers/etc/hosts: Permission denied

I changed Permissions for Users to Full Control and I tried disabling my anti-virus. Still not working.

**Comment by zamoose** [comment]
Make sure you're setting your PowerShell/CMD to "Run as Administrator". 

--
Doug Stewart

**Comment by txbredred** [comment]
@zamoose 
Tried that as well as set "Run as Admin" for Notepad and Notepadd++ per instructions from an MS article. Still no success; however, I did do the following:

Reset the Hosts file using MS Fix It Wizard found here: http://support.microsoft.com/kb/972034
Ran vagrant up and voila!

Finally, Success!

My hosts file contained multiple lines inserted by Spybot Search and Destroy which I no longer run or have installed on my computer. Resetting the host file seemed to do the trick. I'm going to undo the other changes just to see if it continues to run correctly. If not then I'll know it was the combination of steps.

Thank you and also to @cfoellmann  @jerome-nelson, all of your suggestions led me to the right solution!
